editor.planx.uk/src/pages/FlowEditor/components/Settings/shared/SettingsForm.tsx

```diff
@@ -0,0 +1,65 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { FormikProps } from "formik";
+import React from "react";
+import EditorRow from "ui/editor/EditorRow";
+import InputGroup from "ui/editor/InputGroup";
+import InputLegend from "ui/editor/InputLegend";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
+
+type SettingsFormProps<TFormikValues> = {
+  legend: string;
+  description: React.ReactElement;
+  input: React.ReactElement;
+  formik: FormikProps<TFormikValues>;
+  preview?: React.ReactElement;
+};
+
+export const SettingsForm = <TFormikValues,>({
```

Comment:
Apart from changes to types and an update of the `ErrorWrapper` id, this component is copy/pasted from `/DesignSettings/index.tsx`